### ISSUE TITLE
Accessibility: Page titles should be more descriptive

### ISSUE THREAD

**Issue by astridu** [sul-dlss/happy-heron#1799]
Instead of SDR | Stanford Digital Repository for all the pages, could we rename the page titles to be more descriptive of the page they are on? 

For example: 
Home page: SDR | Stanford Digital Repository
Dashboard: SDR | Dashboard
Collections: SDR | Collection or SDR | Collection name (e.g. SDR | Climate Change)
Deposits: SDR | Deposit or SDR | Deposit name (e.g. SDR | Atmospheric Dataset)
